Makefile

```diff
@@ -72,7 +72,7 @@ coverage: upgrade-pip install-backend install-cdkproxy install-tests
 		--color=yes
 
 deploy-image:
-	docker build -f backend/docker/prod/${type}/Dockerfile -t ${image-tag}:${image-tag} . && \
```

Comment:
should this be changed?

Reply:
OMG, sorry

Reply:
back to normal [0d66b88](https://github.com/data-dot-all/dataall/pull/1194/commits/0d66b8856a9c7cb2a57c8920a788eaba61645061)